### PR TITLE
fix(ui5-menu): menu item text doesn't overflow submenu arrow

### PR DESCRIPTION
**Pull request by NHristov-sap**
Fixes: #6795

### CODE REVIEW

File: packages/main/test/pages/Menu.html [hunk 1]
```diff
@@ -110,7 +110,7 @@ <h3 class="header-title">Event logger</h3>
 
 		menu.addEventListener("ui5-before-open", function(event) {
 			const item = event.detail.item;
-			if (item && item.text === "Open" && !fetched) {
+			if (item && item.text.substr(0, 4) === "Open" && !fetched) {
```
Comment (hinzzx) [inline packages/main/test/pages/Menu.html:113]: The change works fine for me, I just think this is redunant.

**Review by hinzzx — APPROVED** [review]

File: packages/main/test/pages/Menu.html (hunk 1 above)
Comment (NHristov-sap) [reply packages/main/test/pages/Menu.html:113]: Hi @hinzzx ,

It is not, as I modified little bit this item text in order to be longer.

BR,
Nikolay

**Review by ilhan007 — CHANGES_REQUESTED** [review]
Please be careful about using asymmetric paddings, margins. In RTL you will to handle them by adding addition CSS like, normal mode
   .something { padding-right: 1rem} 
 and then for rtl
   .something[dir="rtl"] { padding-left: 1rem} 
to mirror ir properly in RTL. 

CSS has good solution to that for a long time, and we always put this as a comment in code reviews - use [CSS Logical props](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values)
In your case padding-right can be replaced with padding-inline-end.

Please have this in mind in future changes

File: packages/main/src/themes/Menu.css [hunk 2]
```diff
@@ -118,4 +126,15 @@
...
+
+.ui5-menu-item-text {
+	overflow: hidden;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+	pointer-events: none;
+}
+
+.ui5-menu-item-text:has(.ui5-menu-item-submenu-icon) {
+	padding-right: 1rem;
```
[the remark this answers is not in the document]
Comment (NHristov-sap) [reply packages/main/src/themes/Menu.css:139]: Yes, you are correct... fixed

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**